meeting-notes/2018/2018-11-19--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,197 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 November 19, 2018
+- **Lead:** @eingenito
+- **Notetaker:** @momack2
+- **Attendees:**
+  - @aschmahmann
+  - @mgoelzer
+  - @kevina
+  - @bigs
+  - @stebalien
+  - @alanshaw
+  - @michaelavila
+  - @hannahhoward
+  
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Announcements
+- @magik6k's update on datstore benchmarking
+- Round of updates
+  - High-priority updates. Are you blocked? On what and whose help do you need.
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+
+
+## Notes
+
+#### Announcements
+
+- OKRs still need projections from some team members: https://docs.google.com/spreadsheets/d/139lROP7-Ee4M4S7A_IO4iIgSgugYm7dct620LYnalII/edit?ts=5bef213d#gid=1720716278. Check the spreadsheet for your OKR items and review the mid-Q completion estimate, and provide an estimate for completion by the end of the quarter.
+
+- Weekly sync changes; make time each week for reviewing progress of the three critical go-ipfs efforts; data transport, providing, and data management.
+  - going to center this meeting more around our 3 main endeavors (starting with datastore)
+  
+
+- Reflection issue - improving how we work: https://github.com/ipfs/go-ipfs/issues/5781. Read it over and please add your comments. Let's not worry about solutions yet - we'll get to that. 
+
+- @stebalien's 'Not blocked by steb' issue. Read it over - there's a new label, and new informaiton about how to reach Steven when you need him.
+
+#### Discussion
+- Datastore benchmarking update from @magik6k
+  - beautiful beautiful graphs that tell us lots more information
+  - would love to see them with an additional 0 for size
+  - jfs is broken
+  - using the default driver (ntfs doesn't have great performance over time)
+  - interested to see graphs with different data sets (expect to see this behave differently on different configurations of data - many small files, large files, etc)
+  - plan: expand to run more benchmarks on more computers to integrate with AWS to auto-run more stuff there.
+
+### Week Update
+
+@name
+  - Done:
+  - Blocked:
+  - Next:
+
+@marten-seemann
+ - Done: 
+  * a lot of review of PRs for the QUIC spec
+  * more work on draft-17
+  * started working on a benchmarking suite for quic-go
```

Comment:
Links?

Reply:
It's still pretty basic: https://github.com/marten-seemann/quic-go-benchmarks
I will have to work on better tooling to get a better picture of what's going on. Developing this tooling and identifying bottlenecks will probably be one of my projects for Q1 2019.